CMakeLists.txt

```diff
@@ -0,0 +1 @@
+/opt/ros/kinetic/share/catkin/cmake/toplevel.cmake
```

Comment:
I would not check this file into the repo (it's auto generated) (same with .catkin_workspace above)